### PR TITLE
Add `message-transact` back

### PR DESCRIPTION
**Pull request by AsceticBear**
[The mock example](https://github.com/darwinia-network/darwinia-2.0/blob/bear-message-transact/pallets/message-transact/src/mock.rs#L159-L214) shows how to integrate with the  `darwinia--messages-substrate` pallets.

### CODE REVIEW

File: runtime/pangolin/src/pallets/mod.rs [hunk 1]
```diff
@@ -100,3 +100,5 @@ mod evm;
 pub use evm::*;
 
 mod base_fee;
+
+mod message_transact;
```
Comment (jiguantong) [inline runtime/pangolin/src/pallets/mod.rs:104]: Need an empty line

**Comment by HackFisher** [comment]
@AsceticBear Please help resolve the conflicts, just merged #79 #70

**Comment by AsceticBear** [comment]
> @AsceticBear Please help resolve the conflicts, just merged #79 #70

@HackFisher Please take a review again.

**Review by HackFisher — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Planning review.
cc @AurevoirXavier

**Pull request merged**